regression-tests/test-results/clang-12-c++20/pure2-last-use.cpp.output

```diff
@@ -23,52 +32,33 @@ pure2-last-use.cpp2:348:193: error: use of address-of-label extension outside of
-                                                                                            ^
-pure2-last-use.cpp2:773:156: error: expected unqualified-id
-auto issue_888_1([[maybe_unused]] std::string unnamed_param_1, std::move_only_function<int([[maybe_unused]] cpp2::impl::in<int> unnamed_param_1)> size) -> void;
-                                                                                                                                                           ^
-pure2-last-use.cpp2:773:160: error: expected '>'
-auto issue_888_1([[maybe_unused]] std::string unnamed_param_1, std::move_only_function<int([[maybe_unused]] cpp2::impl::in<int> unnamed_param_1)> size) -> void;
-                                                                                                                                                               ^
-pure2-last-use.cpp2:773:87: note: to match this '<'
-auto issue_888_1([[maybe_unused]] std::string unnamed_param_1, std::move_only_function<int([[maybe_unused]] cpp2::impl::in<int> unnamed_param_1)> size) -> void;
-                                                                                      ^
-pure2-last-use.cpp2:773:160: error: expected ')'
-auto issue_888_1([[maybe_unused]] std::string unnamed_param_1, std::move_only_function<int([[maybe_unused]] cpp2::impl::in<int> unnamed_param_1)> size) -> void;
-                                                                                                                                                               ^
-pure2-last-use.cpp2:773:17: note: to match this '('
-auto issue_888_1([[maybe_unused]] std::string unnamed_param_1, std::move_only_function<int([[maybe_unused]] cpp2::impl::in<int> unnamed_param_1)> size) -> void;
-                ^
-pure2-last-use.cpp2:271:7: error: missing '}' at end of definition of 'issue_857_4'
-class issue_857_4 {
-      ^
-pure2-last-use.cpp2:905:1: note: still within definition of 'issue_857_4' here
-namespace captures {
-^
 pure2-last-use.cpp2:279:272: error: no member named 'move_only_function' in namespace 'std'
-requires (std::is_convertible_v<CPP2_TYPEOF(f_), std::add_const_t<std::add_pointer_t<int()>>&> && std::is_convertible_v<CPP2_TYPEOF(g_), std::add_const_t<std::add_pointer_t<int(cpp2::impl::in<int> in_)>>&> && std::is_convertible_v<CPP2_TYPEOF(mf_), std::add_const_t<std::move_only_function<int()>>&> && std::is_convertible_v<CPP2_TYPEOF(mg_), std::add_const_t<std::move_only_function<int(cpp2::impl::in<int> in_)>>&>) 
+requires (std::is_convertible_v<CPP2_TYPEOF(f_), std::add_const_t<std::add_pointer_t<int()>>&> && std::is_convertible_v<CPP2_TYPEOF(g_), std::add_const_t<std::add_pointer_t<int(cpp2::impl::in<int> inx)>>&> && std::is_convertible_v<CPP2_TYPEOF(mf_), std::add_const_t<std::move_only_function<int()>>&> && std::is_convertible_v<CPP2_TYPEOF(mg_), std::add_const_t<std::move_only_function<int(cpp2::impl::in<int> inx)>>&>) 
```

Comment:
This seems to be one of the lowering bugs mentioned at <https://github.com/hsutter/cppfront/commit/212eb15e77bba222c7fda3ff50f1e5f479a627ee#commitcomment-147040717>.